### ISSUE TITLE
Asking the user about an update should be only once per version, not every SE site.

### ISSUE THREAD

**Issue by rschrieken** [SO-Close-Vote-Reviewers/UserScripts#84]
When we (or really I) push a new version of the script, users get the confirm box pushed in their face:

https://github.com/SO-Close-Vote-Reviewers/UserScripts/blob/master/SECloseVoteRequestGenerator.user.js#L86

Can we at least make this less blocking?

Reported several times in the past but recently by FerryBig: http://chat.stackoverflow.com/transcript/message/34171354#34171354

**Comment by makyen** [comment]
Thanks for the report. :) 

As far as I'm aware, this was resolved by switching the repository over to the 2.X version of the Request Generator, which uses userscript storage for this information, rather than per-site `localStorage`. If it's still an issue, please post a new comment here, reopen this issue, or create a new issue.
